src/metapath-results.js

```diff
@@ -106,7 +106,18 @@ class TableFull extends Component {
 
   // compare numbers (for sorting)
   compareNumbers(a, b, key) {
-    return Number(a[key]) - Number(b[key]) || a[key] - b[key];
+    // parse as numbers
+    const comparison = Number(a[key]) - Number(b[key]);
+    if (comparison)
```

Comment:
This is still a little problematic. When `a[key]` and `b[key]` are both valid numbers and they are equal, the function should return 0 at this point, but it actually executes the code on lines 114-120 (fortunately the return value is still correct). 

I suppose your logic is as follows: when `a[key]` and `b[key]` are or can be converted into valid numbers, compare them numerically; otherwise compare them as strings. I guess a more explicit way to implement this logic is to change this line into:
``` 
if (!Number.isNaN(comparison))
```

Reply:
The reason I did it this way was because `Number(null)` returns 0 for some reason, so I wanted to catch that with a falsey check.